src/Sparrow/Json/BlittableJsonReaderObject.cs

```diff
@@ -575,7 +575,7 @@ public int GetPropertyIndex(StringSegment name)
             long metadataSize = currentOffsetSize + currentPropertyIdSize + sizeof(byte);
             byte* metadataPtr = _metadataPtr;
 
-            int mid = comparer.LastFoundAt ?? (min + max) / 2;
```

Comment:
This is actually important optimization. Why not have it?

Reply:
Nobody ever sets the value so it's always null.